docs/sources/vendor/Cisco/cisco_meraki.md

```diff
@@ -1,65 +1,96 @@
 ## Meraki (MR, MS, MX)
 
 ## Key facts
-* In most cases, Cisco Meraki logs are general and require vendor product by source configuration.
-* For distinctive log messages, filters are based on the appliance name and program value.
+* Cisco Meraki messages are not distinctive, which means that it's impossible to parse the sourcetype based on the log message.
+* Because of the above you should either configure known Cisco Meraki hosts in SC4S, or open unique ports for Cisco Meraki devices.
+* [Splunk Add-on for Cisco Meraki 2.1.0](https://splunkbase.splunk.com/app/5580) doesn't support syslog. Use [TA-meraki](https://splunkbase.splunk.com/app/3018) instead. `TA-meraki 1.1.5` requires sourcetype `meraki`.
 
-## Distinctive log messages
-See samples in the [vendor documentation](https://documentation.meraki.com/General_Administration/Monitoring_and_Reporting/Syslog_Event_Types_and_Log_Samples).
-
-The two conjuncted conditions are required:
-
-1. Program: `(events|urls|firewall|cellular_firewall|vpn_firewall|ids-alerts|flows)`
-
-2. Appliance name:
-
-| Sourcetype | Distinct element |
-| ---------  | --------------   |
-| meraki:accesspoints | `host('MR' type(string) flags(ignore-case,prefix))` |
-| meraki:securityappliances | `host('MX' type(string) flags(ignore-case,prefix))` |
-| meraki:switches | `host('MS' type(string) flags(ignore-case,prefix))` |
- 
 
 ## Links
-
 | Ref            | Link                                                                                                    |
 |----------------|---------------------------------------------------------------------------------------------------------|
-| Splunk Add-on  | <https://splunkbase.splunk.com/app/5580>                                                                 |
-| Product Manual | <https://documentation.meraki.com/zGeneral_Administration/Monitoring_and_Reporting/Syslog_Server_Overview_and_Configuration> <https://documentation.meraki.com/General_Administration/Monitoring_and_Reporting/Syslog_Event_Types_and_Log_Samples> |
+| Splunk Add-on  | <https://splunkbase.splunk.com/app/3018>                                                                 |
```

Comment:
@mstopa-splunk @rjha-splunk it's not Splunk supported add-on, not sure that we need to do it in general